charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/auto-install: fleet-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.27.0-0'
```

Comment:
Shouldnt this include 1.27?

Reply:
Might that be a separate change from this PR? AFAICT this is just about unblocking QA.

Reply:
This was my previous mistake, this was solved on the new push